--- a/socket/udp-turn.c	2015-04-21 05:37:13.000000000 +0800
+++ b/socket/udp-turn.c	2017-02-20 15:24:33.024922025 +0800
@@ -235,7 +235,7 @@
           priv_send_data_queue_destroy);
 
   sock->type = NICE_SOCKET_TYPE_UDP_TURN;
-  sock->fileno = base_socket->fileno;
+  sock->fileno = NULL;//base_socket->fileno;
   sock->addr = *addr;
   sock->send_messages = socket_send_messages;
   sock->send_messages_reliable = socket_send_messages_reliable;
