--- a/stun/usages/timer.h	2017-02-22 03:46:57.000000000 +0800
+++ b/stun/usages/timer.h	2017-02-22 09:18:01.772737207 +0800
@@ -134,14 +134,14 @@
  * cases as it is also what is used by SIP style VoIP when sending A-Law and
  * mu-Law audio, so 200ms should be hyper safe.
  */
-#define STUN_TIMER_DEFAULT_TIMEOUT 200
+#define STUN_TIMER_DEFAULT_TIMEOUT 600
 
 /**
  * STUN_TIMER_DEFAULT_MAX_RETRANSMISSIONS:
  *
  * The default maximum retransmissions allowed before a timer decides to timeout
  */
-#define STUN_TIMER_DEFAULT_MAX_RETRANSMISSIONS 7
+#define STUN_TIMER_DEFAULT_MAX_RETRANSMISSIONS 3
 
 /**
  * STUN_TIMER_DEFAULT_RELIABLE_TIMEOUT:
